.travis.yml

```diff
@@ -0,0 +1,15 @@
+env:
+  global:
+    - TEST_TIMEOUT_SCALE=20 # Set higher timeouts for Travis
+    - PACKAGE=github.com/$TRAVIS_REPO_SLUG
+    - CHANGE_MINIKUBE_NONE_USER=true
+
+dist: trusty
```

Comment:
Would this work with a newer distro? Seems trusty is kinda old: https://wiki.ubuntu.com/Releases

Reply:
Sadly it's the highest version they [offer](https://docs.travis-ci.com/user/reference/overview/)